jalali/10-dey.yml

```diff
@@ -37,7 +37,7 @@ events:
     - Ancient Iran
     fa_IR:
     - باستانی
-- partial_key: "220"
+- partial_key: "earthquake_safety_day"
   title:
     fa_IR: روز ایمنی در برابر زلزله و کاهش اثرات بلایای طبیعی
   description:
```

Comment:
Please specified the first "Year" that held in calendar.
year : 1358

Reply:
done